stack-9.0.1.yaml

```diff
@@ -16,6 +16,7 @@ packages:
   - ./plugins/hls-explicit-imports-plugin
   # - ./plugins/hls-refine-imports-plugin
   - ./plugins/hls-hlint-plugin
+  # - ./plugins/hls-rename-plugin
```

Comment:
is the rename plugin not compatible with 9.0? What's blocking it?

Reply:
There were just some small issues. It is now working